src/main/java/com/training/assignments/Assignment1.java

```diff
@@ -0,0 +1,35 @@
+package com.training.assignments;
+
+import java.util.Scanner;
+
+public class Assignment1 {
+    public double intToDouble(int intVal) {
```

Comment:
Give some meaningful name intVal--> value 
please apply this for below code as well.